pages/deployments/oidc/oidc_with_aws.md

```diff
@@ -0,0 +1,204 @@
+        - `BUILD_COMMIT` (optional) can be omitted and if so, is usually replaced with a single wildcard `*` at the end of the line.
+        - `STEP_KEY` (optional) can be omitted and if so, is usually replaced with a single wildcard `*` at the end of the line.
+
+    **Note:** When formulating your _subject_ field's value, you can replace any of the constituent field values above with a wildcard `*` to not set limits on those constituent fields.
+
+    You can also allow this IAM role to be used with other pipelines, branches, commits and steps by specifying multiple comma-separated values for the `agent.buildkite.com:sub` _subject_ field.
+
+    For example, to allow the IAM role to be used for any pipeline in the Buildkite organization `example-org`, when building on any branch or tag, specify a single _subject_ field value of `organization:example-org:pipeline:*:ref:refs:*`.
+
+1. Modify the `Condition` section's `IpAddress` values (`AGENT_PUBLIC_IP_ONE` and `AGENT_PUBLIC_IP_TWO`) with a list of your agent's IP addresses.
+
+    Only OIDC token exchange requests (for IAM roles) from Buildkite Agents with these IP addresses will be permitted.
+
+1. Verify that your custom trust policy is complete. The following example trust policy (noting that `AWS_ACCOUNT_ID` has not been specified) will only allow the exchange of an agent's OIDC tokens with IAM roles when:
+    * the Buildkite organization is `example-org`
+    * building on both the `main` branch and tagged releases
+    * on Buildkite Agents whose IP addresses are either `192.0.2.0` or `198.51.100.0`
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Federated": "arn\:aws\:iam:\:AWS_ACCOUNT_ID\:oidc-provider/agent.buildkite.com"
+                },
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Condition": {
+                    "StringEquals": {
+                        "agent.buildkite.com:aud": "sts.amazonaws.com"
+                    },
+                    "StringLike": {
+                        "agent.buildkite.com:sub": [
+                            "organization\:example-org\:pipeline\:example-pipeline\:ref:refs/heads/main:*",
+                            "organization\:example-org\:pipeline\:example-pipeline\:ref:refs/tags/*:*"
+                        ]
+                    },
+                    "IpAddress": {
+                        "aws:SourceIp": [
+                            "192.0.2.0",
+                            "198.51.100.0"
+                        ]
+                    }
+
+                }
+            }
+        ]
+    }
+    ```
+
+1. In the **Custom trust policy** section, copy your modified custom trust policy, paste it into your IAM role, and complete the next few steps up to specifying the **Role name**.
```

Comment:
```suggestion
1. In the **Custom trust policy** section, copy your modified custom trust policy, paste it into your IAM role, and complete the next few steps up to specify the **Role name**.
```

Reply:
From a grammatical perspective, I think 'specify' should be kept in its gerund form.